packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -160,6 +160,15 @@ export const LivenessCameraModule = (
     videoWidth && videoHeight ? videoWidth / videoHeight : 0
   );
 
+  // Only allow on mobile screen if it is no light challenge
+  const allowSelectableDevicesMobile =
+    !isMobileScreen || isFaceMovementChallenge;
+  const allowSelectableDevices =
+    isStartView &&
+    allowSelectableDevicesMobile &&
```

Comment:
If i'm reading correctly we only allow device selection for mobile?

Reply:
We allow it on mobile only if it is a `FaceMovementChallenge` only (no light)